src/main/java/ch/wisv/areafiftylan/controller/CurrentUserRestController.java

```diff
@@ -129,6 +124,19 @@
     }
 
     /**
+     * Get the tickets of the currently logged in user. All the tickets owned by this user will be returned.
+     *
+     * @param auth The current User, injected by spring
+     *
+     * @return The current owned tickets, if any exist
+     */
+    @RequestMapping(value = "/tickets", method = RequestMethod.GET)
+    public Collection<Ticket> getAllTickets(Authentication auth) {
```

Comment:
Why do we use List and Collection as return types in the CurrentUserRestController? Maybe it's better if we just use one return type.